packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SentenceScopeHandler.ts

```diff
@@ -0,0 +1,46 @@
+import { Direction, Range } from "@cursorless/common";
+import { imap } from "itertools";
+import { NestedScopeHandler } from ".";
+import { trimRange } from "../../../util/rangeUtils";
+import { TokenTarget } from "../../targets";
+import type { TargetScope } from "./scope.types";
+
+export default class SentenceScopeHandler extends NestedScopeHandler {
+  public readonly scopeType = { type: "sentence" } as const;
+  public readonly iterationScopeType = { type: "paragraph" } as const;
+
+  protected generateScopesInSearchScope(
+    direction: Direction,
+    { editor, domain }: TargetScope,
+  ): Iterable<TargetScope> {
+    const segmenter = new Intl.Segmenter("en", { granularity: "sentence" });
```

Comment:
Should we use the user locale instead of hardcoding `en`?

Reply:
We could just pass undefined or `best fit`

Reply:
fixed